tools/doc/json.js

```diff
@@ -64,8 +64,8 @@ function doJSON(input, filename, cb) {
     if (type === 'heading' &&
         !text.trim().match(/^example/i)) {
       if (tok.depth - depth > 1) {
-        return cb(new Error('Inappropriate heading level\n' +
-                            JSON.stringify(tok)));
+        return cb(new Error(`Inappropriate heading level
+${JSON.stringify(tok)}`));
```

Comment:
Nit - we actually prefer not to use multiline template strings. This should likely be changed while landing and we might think about adding a eslint rule against this as it comes up more often.

Reply:
The same applies to the other template strings in here.